### PR TITLE
[enterprise-metrics] Bump GEM image version

### PR DESCRIPTION
**Pull request by dimitarvdimitrov**
* updates the version of the docker image for GEM to 2.0.1

### CODE REVIEW

File: enterprise-metrics/CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ Entries should include a reference to the Pull Request that introduced the chang
   - ruler_storage.backend used to be s3 but is now filesystem
   - alertmanager_storage.backend used to be s3 but is now filesystem
   - See [GEM 2.0 release notes](https://grafana.com/docs/enterprise-metrics/latest/release-notes/v2-0/) for more information
+- [FEATURE] Upgrade to [Grafana Enterprise Metrics v2.0.1](https://grafana.com/docs/metrics-enterprise/latest/downloads/#v200----april-13th-2022). #767
```
Comment (Logiraptor) [inline enterprise-metrics/CHANGELOG.md:38]: Just need to update this link to point to the correct header for v2.0.1 once it exists

**Review by Logiraptor — APPROVED** [review]
Unblocking with minor feedback

File: enterprise-metrics/CHANGELOG.md (hunk 1 above)
Comment (dimitarvdimitrov) [reply enterprise-metrics/CHANGELOG.md:38]: I thought these were the release notes... mb

**Review by Logiraptor — APPROVED** [review]

File: enterprise-metrics/CHANGELOG.md (hunk 1 above)
Comment (dimitarvdimitrov) [reply enterprise-metrics/CHANGELOG.md]: will merge only after deploying the website

**Comment by dimitarvdimitrov** [comment]
I also updated other links in the CHANGELOG. The metrics-enterprise->enterprise-metrics redirect loses the anchor at the end of the URL, so the link to 1.4.0 for example just reidrected you to the top of the downloads page on the latest version of the docs

**Pull request merged**